modyn/trainer_server/internal/trainer/pytorch_trainer.py

```diff
@@ -452,7 +287,7 @@ def train(self) -> None:  # pylint: disable=too-many-locals, too-many-branches
                         loss = torch.dot(self._criterion_nored(output, target), weights / weights.sum())
                     else:
                         loss = self._criterion(output, target)
-                stopw.stop("Loss")
```

Comment:
I added a level of indentation here, I suppose that's fair?

Reply:
Yes, probably it will not make a difference, unless exiting the amp scope for some reason is an expensive operation. Good catch!